File: task-03/ruby/subtask-2.rb
def subtask2
  content = File.read('input3.txt')
  File.write('output3.txt', content)
  final = File.read('output3.txt')
  return final
end

print "Content of the file: \n"
inputcontent = gets

File.write('input3.txt', inputcontent)

puts "printing output.txt..."
result = subtask2
puts result

File: task-03/python/subtask-2.py
#Write a program that reads a string from a file named input.txt and writes that string to another file named output.txt

def subtask2():
    with open('input.txt', 'r') as infile:
        content = infile.read()
    with open('output.txt', 'w') as outfile:
        outfile.write(content)
    with open('output.txt', 'r') as outfile:
        final = outfile.read()
    return final

inputcontent = input("Content of the file: \n")

with open('input.txt', 'w') as infile:
    infile.write(inputcontent)


print("printing output.txt...")
result = subtask2()
print(result)
